Sources/UtilityBeltUITesting_XCTestSupport/Extensions/XCUIApplication+LaunchArguments.swift

```diff
@@ -0,0 +1,24 @@
+// Copyright © 2021 SpotHero, Inc. All rights reserved.
+
+#if canImport(XCTest)
+
+    import UtilityBeltUITesting
+    import XCTest
+
+    public extension XCUIApplication {
+        /// Convenience method for merging new launch arguments into the launch arguments array.
+        /// - Parameter objects: An array of launch argument objects that will be adding into the launch arguments.
+        func appendLaunchArguments(_ arguments: [LaunchArgument]) {
+            arguments.forEach { argument in
+                switch argument {
+                case .custom:
+                    self.launchArguments.append(argument.formattedKey)
+                case let .argument(_, value),
+                     let .userDefault(_, value):
+                    self.launchArguments.append(contentsOf: [argument.formattedKey, value])
```

Comment:
does this matter if its one string like "-key value" or two strings like "-key" and "value"? I was under the impression each string represented a full argument, but it looks like the key and value are each appended on the array here. It would certainly be easier to parse these back into the enum type if each string represented a full argument.

Reply:
This would be handled through https://spothero.atlassian.net/browse/IOS-2719

Reply:
I've opted to separate out the proper command line arguments and user default code to that card.